### PR TITLE
added function examples via emoji_pet

### PR DESCRIPTION
**Pull request by octaflop**
some more rudimentary :smile: :cry: examples and examples of using inputs and functions to run small-to-medium-sized programs.

### CODE REVIEW

File: examples/emoji_pet.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+grin_face = Image("09090:00000:90009:09990:00000")
+happy_face = Image("09090:00000:99999:90009:09990")
+
+frowning_face = Image("09090:00000:09990:90009:00000")
+sad_face = Image("09090:00000:09990:90009:90009")
+
+small_tongue = Image("07090:00000:99999:00990:00000")
+
+
+def animate(faces, delay=500):
```
Comment (markshannon) [inline examples/emoji_pet.py]: `display.show()`  accepts an iterable, so this is unnecessary

File: examples/emoji_pet.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    for face in faces:
+        display.show(face)
+        sleep(delay)
+
+
+def get_happy(loops=2):
+    # A series of happy faces
+    music.play(music.BA_DING, wait=False)
+    for i in range(loops):
+        animate([neutral_face, grin_face, happy_face])
```
Comment (markshannon) [inline examples/emoji_pet.py]: You can replace this with `display.show([neutral_face, grin_face, happy_face], delay = 500)`

File: examples/emoji_pet.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+def get_sad(loops=2):
+    # A series of sad faces
+    music.play(music.POWER_DOWN, wait=False)
+    for i in range(loops):
+        animate([neutral_face, frowning_face, sad_face])
```
Comment (markshannon) [inline examples/emoji_pet.py]: ditto

File: examples/emoji_pet.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+
+def get_neutral(loops=2):
+    # A series of grim / neutral faces
+    for i in range(loops):
+        animate([
```
Comment (markshannon) [inline examples/emoji_pet.py]: and here as well...

**Comment by octaflop** [comment]
Should have had a better look at the docs! Fixed to not use the `animate` function and use the built-in tools.

File: examples/emoji_pet.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+grin_face = Image("09090:00000:90009:09990:00000")
+happy_face = Image("09090:00000:99999:90009:09990")
+
+frowning_face = Image("09090:00000:09990:90009:00000")
+sad_face = Image("09090:00000:09990:90009:90009")
+
+small_tongue = Image("07090:00000:99999:00990:00000")
+
+
+def get_happy(loops=2, delay=500):
```
Comment (ntoll) [inline examples/emoji_pet.py]: What does/did loops do? If it's not used, please remove. ;-)

File: examples/emoji_pet.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    # A series of happy faces
+    music.play(music.BA_DING, wait=False)
+    display.show([neutral_face, grin_face, happy_face], delay=delay)
+
+
+def get_sad(loops=2, delay=500):
```
Comment (ntoll) [inline examples/emoji_pet.py]: Ditto loops.

File: examples/emoji_pet.py [hunk 7]
```diff
@@ -0,0 +1,66 @@
+"""
+    emoji_pet.py
+    ~~~~~~~~
+
+    A reactive "pet" which changes face depending on certain inputs
+    editable by the user. Emotions are functions of an animation function.
```
Comment (ntoll) [inline examples/emoji_pet.py]: Perhaps, "Emotions are represented by animations that use the display.show function"..?

File: examples/emoji_pet.py [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    display.show([neutral_face, grin_face, happy_face], delay=delay)
+
+
+def get_sad(loops=2, delay=500):
+    # A series of sad faces
+    music.play(music.POWER_DOWN, wait=False)
+    display.show([neutral_face, frowning_face, sad_face], delay=delay)
+
+
+def get_neutral(loops=2, delay=500):
```
Comment (ntoll) [inline examples/emoji_pet.py]: Ditto loops

File: examples/emoji_pet.py [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+    # A series of grim / neutral faces
+    display.show([
+        neutral_face2,
+        squiggle_face1,
+        squiggle_face2,
+        neutral_face
+    ], delay=delay)
+
+
+def get_silly(loops=2, delay=500):
```
Comment (ntoll) [inline examples/emoji_pet.py]: You guessed it, loops...!?!?! ;-)

**Comment by ntoll** [comment]
Can you fix the surplus `loops` args and squash into a single commit rebased with master..? ;-)

**Comment by octaflop** [comment]
Haven't done a rebase for a couple of years, but I think I finally got it up to snuff — I may have had to [pull out the big guns](https://xkcd.com/1597/) a couple of times.

Thanks for your patience and review. Lemme know.

**Pull request closed without merging**